doc/specs/#2325 - Default Profile Settings.md

```diff
@@ -0,0 +1,314 @@
+---
+author: Mike Griese @zadjii-msft
+created on: 2019-11-13
+last updated: 2019-11-13
+issue id: #2325
+---
+
+# Default Profile Settings
+
+## Abstract
+
+Oftentimes, users have some common settings that they'd like applied to all of
+their profiles, without needing to manually edit the settings of each of them.
+This doc will cover some of the many proposals on how to expose that
+functionality to the user in our JSON settings model. In this first draft, we'll
+examine a number of proposed solutions.
+
+## Inspiration
+
+During the course of the pull request review on [#3369], the original pull request for this feature's implementation, it became apparent that the entire team has differing opinions on how this feature should be exposed to the user. This doc is born from that discussion.
+
+## Solution Proposals
+
+### Proposal 1: `defaultSettings` Profile object in the global settings
+
+```json
+{
+    "$schema": "https://aka.ms/terminal-profiles-schema",
+    "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
+    "defaultSettings":
+    {
+        "useAcrylic": true,
+        "acrylicOpacity": 0.1,
+        "fontFace": "Cascadia Code",
+        "fontSize": 10
+    },
+    "requestedTheme" : "dark",
+    "showTabsInTitlebar" : true,
+    "profiles":
+    [
+        {
+            "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
+            "name": "Windows PowerShell",
+            "commandline": "powershell.exe",
+            "hidden": false
+        },
+        {
+            "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
+            "name": "cmd",
+            "commandline": "cmd.exe",
+            "hidden": false
+        }
+    ],
+    "schemes": [],
+    "keybindings": []
+}
+```
+#### Benefits
+
+##### Clearly encapsulates the default profile settings
+Puts all the default profiles settings in one object. It's immediately obvious
+when scanning the file where the defaults are.
+
+##### Simple to understand
+There's one object that applies to all the subsequent profiles, and that
+object is the `defaultSettings` object.
+
+#### Concerns
+
+##### What do we name this setting?
+People were concerned about the naming of this property. No one has a name that
+we're quite happy with:
+
+* `defaultSettings`: This kinda seems to conflict conceptually with
+  "defaults.json". It's different, but is that obvious?
+* `defaultProfileSettings`: Implies "settings of the default profile"
+* `defaults`: This kinda seems to conflict conceptually with "defaults.json"
+* `baseProfileSettings`: not the worst, but not terribly intuitive
+* Others considered with less enthusiasm
+    - `profiles.defaults`: people don't love the idea of a `.`, but hey, VsCode does it.
+    - `inheritedSettings`
+    - `rootSettings`
+    - `globalSettings`: again maybe conflicts a bit with other concepts/properties
+    - `profileSettings`
+    - `profilePrototype`

```

Comment:
I do not consider a conflict with defaults.json to be a reason to not do a thing. If the problem is one of user education, we can get the word out on the streets in a different way.

Reply:
I strongly agree.